examples/func_types/canisters/func_types/func_types.did

```diff
@@ -67,4 +67,7 @@ service : () -> {
     ) query;
```

Comment:
Have we tested null as a param? If that doesn't work, then I think we should add a test case